changelog/unreleased/remove-proxy-route-fix-cs3-example.md

```diff
@@ -0,0 +1,5 @@
+Bugfix: Remove non working proxy route and fix cs3 users example
+
+We added removed a non working route from the proxy default configuration and fixed the cs3 users deployment example since it still used the accounts service.
```

Comment:
```suggestion
We removed a non working route from the proxy default configuration and fixed the cs3 users deployment example since it still used the accounts service. It now [...]
```